test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -824,6 +826,43 @@ def create_pytorch_only_extra_kwargs():
             create_pytorch_only_extra_kwargs,
         )
 
+
+def _parameterized_class_attrs_and_values_with_fake_options():
+    input_values = []
+    input_values.extend(
+        itertools.product((True, False), (True, False), (True, False), (True, False))
+    )
+    return {
+        "attrs": [
+            "op_level_debug",
+            "dynamic_shapes",
+            "load_checkpoint_during_init",
+            "export_within_fake_mode",
+        ],
+        "input_values": input_values,
+    }
+
+
+@parameterized.parameterized_class(
+    **_parameterized_class_attrs_and_values_with_fake_options(),
+    class_name_func=_parameterize_class_name,
+)
+class TestFxToOnnxFakeTensorWithOnnxRuntime(onnx_test_common._TestONNXRuntime):
+    """ONNX export test for specific Fake Tensor scenarios
+
+    TODO: Should we merge this with  `TestFxToOnnxWithOnnxRuntime`? Considerably increases export time
```

Comment:
is it because of two new flags thus 4x time?

Perhaps just add one scenario for now to cover the fake mode feature and that it plays nicely with others. This can be followup pr. IIRC the tests are pretty light and fast so it is not that bad.